stories/tree/tree.stories.js

```diff
@@ -1,897 +1,1019 @@
+                                </ul>
+                            </li>
+                            <li role="treeitem" aria-level="4" class="fd-tree__item">
+                                <div tabindex="0" class="fd-tree__item-container">
+                                    <a href="https://sap.github.io/fundamental/" class="fd-tree__content has-navigation-indicator">
+                                        <span class="fd-tree__text">Level 4</span>
+                                    </a>
                                 </div>
                             </li>
                         </ul>
                     </li>
+                    <li role="treeitem" aria-level="3" class="fd-tree__item">
+                        <div tabindex="0" class="fd-tree__item-container">
+                            <a href="https://sap.github.io/fundamental/" class="fd-tree__content has-navigation-indicator">
+                                <span class="fd-tree__text">Level 3</span>
+                            </a>
+                        </div>
+                    </li>
                 </ul>
             </li>
-            <li class="fd-tree__item" role="treeitem">
-                <div class="fd-tree__row">
-                    <div class="fd-tree__col fd-tree__col--control">
-                        Second Level
+            <li role="treeitem" aria-level="2" class="fd-tree__item">
+                <div tabindex="0" class="fd-tree__item-container">
+                    <div class="fd-tree__content">
+                        <span class="fd-tree__text">Level 2</span>
                     </div>
-                    <div class="fd-tree__col">
-                        Secondary Value
+                </div>
+            </li>
+        </ul>
+    </li>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container has-highlight-indicator--success is-navigated">
+            <a href="https://sap.github.io/fundamental/" class="fd-tree__content has-navigation-indicator">
+                <span class="fd-tree__text">Level 1</span>
+            </a>
+        </div>
+    </li>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container has-highlight-indicator--success">
+        <a href="https://sap.github.io/fundamental/" class="fd-tree__content has-navigation-indicator">
+            <span class="fd-tree__text">Level 1</span>
+        </a>
+        </div>
+    </li>
+</ul>
+`;
+
+treeWithExpandedLevel6AndNavigationLinks.parameters = {
+    docs: {
+        iframeHeight: 1350,
+        storyDescription: `
+When Level 6 is expanded to show up to 12 levels:
+
+Indentations are calculated with 0.25rem
+
+- Level 2 Changes to be indented by 0.25rem
+- Level 3 Changes to be indented by 0.5rem
+- Level 4 Changes to be indented by 0.75rem
+- Level 5 Changes to be indented by 1rem
+- Level 6 Changes to be indented by 1.25rem
```

Comment:
Same thing here: "Level 2 indents by 0.25rem," otherwise the "changes to" complicates the sentence.